docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.2'
```

Comment:
Why the docker-compose file?

Reply:
used by travis
We may not need it with simple vanilla elixir setup... but I like the flexability